packages/presentation/lib/src/pages/movie_details_page/widgets/reviews_widget.dart

```diff
@@ -0,0 +1,165 @@
+      padding: const EdgeInsets.all(Dimens.size18),
+      width: double.infinity,
+      height: MediaQuery.of(context).size.height - Dimens.size250,
+      child: isLoading
+          ? ListView(
+              children: List.generate(
+                AppConst.shadowMovieListLength,
+                (index) => ReviewListItemShadow(),
+              ),
+            )
+          : ListView.builder(
+              itemCount: reviews.length,
+              itemBuilder: (_, index) => ReviewListItem(item: reviews[index]),
+            ),
+    );
+  }
+}
+
+class ReviewListItem extends StatelessWidget {
+  const ReviewListItem({
+    required this.item,
+    super.key,
+  });
+
+  final ReviewMessageUi item;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.start,
+      children: [
+        MessageField(
+          message: item.message,
+          starsCount: item.starsCount,
+        ),
+        AvatarWithName(
+          imageUrl: item.imageUrl,
+          date: item.date,
+          userName: item.userName,
+        )
+      ],
+    );
+  }
+}
+
+class MessageField extends StatelessWidget {
+  const MessageField({
+    required this.message,
+    required this.starsCount,
+    super.key,
+  });
+
+  final int starsCount;
+  final String message;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+        color: Color(MyColors.messageBackground),
+        borderRadius: BorderRadius.circular(Dimens.size10),
+      ),
+      padding: EdgeInsets.only(
+        left: Dimens.size18,
+        right: Dimens.size18,
+        bottom: Dimens.size18,
+      ),
+      child: ExpandablePanel(
+        header: MovieStars(
+          fullStarsCount: starsCount,
+          isBigStar: false,
+        ),
+        collapsed: Text(
+          message,
+          style: sfProSemiRegular14px70opacity,
+          softWrap: true,
+          maxLines: AppConst.maxDescriptionTextLines,
+          overflow: TextOverflow.ellipsis,
+        ),
+        expanded: Text(
+          message,
+          style: sfProSemiRegular14px70opacity,
+          softWrap: true,
+        ),
+        theme: ExpandableThemeData(
+          hasIcon: true,
+          tapBodyToCollapse: true,
+        ),
+      ),
+    );
+  }
+}
+
+class AvatarWithName extends StatelessWidget {
+  const AvatarWithName({
+    required this.imageUrl,
+    required this.date,
+    required this.userName,
+    super.key,
+  });
+
+  final String? imageUrl;
+  final String userName;
+  final String date;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(Dimens.size18),
+      child: Row(
+        children: [
+          imageUrl != null
+              ? PersonImage(imageUrl: imageUrl!)
+              : ImageErrorWidget(
+                  height: Dimens.size50,
+                  width: Dimens.size50,
+                ),
```

Comment:
up to u, i suggest you use the FadeInImage.assetNetwork widget, advantage will be:
 - placeholder (even when the image hasn't loaded yet);
 - beautiful animation transition after the image has loaded;
 - catching errors.
And all this from just one widget :)
 
example: 
`FadeInImage.assetNetwork(
                placeholder: PlaceholderImage,
                image: Person,
                fit: BoxFit....,
                imageErrorBuilder: (
                  context,
                  error,
                  stackTrace,
                ) {
                  return Image.asset(
                    PlaceholderImage,
                    fit: BoxFit...,
                  );
                },
              ),`

Reply:
> up to u, i suggest you use the FadeInImage.assetNetwork widget, advantage will be:
> 
> * placeholder (even when the image hasn't loaded yet);
> * beautiful animation transition after the image has loaded;
> * catching errors.
>   And all this from just one widget :)
> 
> example: `FadeInImage.assetNetwork( placeholder: PlaceholderImage, image: Person, fit: BoxFit...., imageErrorBuilder: ( context, error, stackTrace, ) { return Image.asset( PlaceholderImage, fit: BoxFit..., ); }, ),`

it's too late =) Mb during "refactoring task"

Reply:
> I find ClipOval better, but it up to u
> 
> <img alt="Screenshot 2022-10-31 at 10 52 20" width="411" src="https://user-images.githubusercontent.com/111364147/198958377-d9f66af0-31c9-4822-910c-fc1eda9256b2.png">

I use ClipRRect in PersonImage() =) it the same